qa/suites/upgrade/octopus-x/parallel/1-ceph-install/octopus.yaml

```diff
@@ -0,0 +1,68 @@
+meta:
+- desc: |
+   install ceph/octopus latest
+   run workload and upgrade-sequence in parallel
+   upgrade the client node
+tasks:
+- install:
+    #exclude_packages:
+      #- ceph-mgr-cephadm
+      #- cephadm
+      #- libcephfs-dev
+    branch: octopus
+- print: "**** done installing octopus"
+- cephadm:
+    roleless: true
+    cephadm_branch: octopus
+    #use image from for example dockerhub
+    #image: docker.io/ceph/daemon-base:latest-octopus-devel
+    #from  https://quay.io/repository/ceph-ci/ceph
+    branch: octopus
+- print: "**** done cephadm: octopus, ceph and daemons in containers setup"
+- cephadm.shell:
+    host.a:
+      - ceph orch status
+      - ceph orch ps
+      - ceph orch ls
+      - ceph orch host ls
+      - ceph orch device ls
+- print: "**** done cephadm.shell"
+- cephadm:
+    log-whitelist:
+      - overall HEALTH_
+      - \(FS_
+      - \(MDS_
+      - \(OSD_
+      - \(MON_DOWN\)
+      - \(CACHE_POOL_
+      - \(POOL_
+      - \(MGR_DOWN\)
+      - \(PG_
+      - \(SMALLER_PGP_NUM\)
+      - Monitor daemon marked osd
+      - Behind on trimming
+      - Manager daemon
+      - Not found or unloadable
+    conf:
+      global:
+        mon warn on pool no app: false
+        bluestore_warn_on_legacy_statfs: false
+        bluestore warn on no per pool omap: false
+        mon pg warn min per osd: 0
+- print: "**** done cephadm"
+- exec:
+    osd.0:
+      - ceph osd set-require-min-compat-client octopus
```

Comment:
can you please make a tracker issue that we verify this in cephadm, before initiating the upgrade to pacific?